netket/machine/jax.py

```diff
@@ -168,7 +165,7 @@ def vector_jacobian_prod(
 
     @property
     def is_holomorphic(self):
-        return self._dtype is complex
+        return True
```

Comment:
I think that this shouldn't be changed (basically because it is used by SR to determine the kind of update), but instead we should always use True in the jacobian only?